jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -298,6 +298,10 @@ listen stats
     acl private src <%= p("ha_proxy.trusted_stats_cidrs") %>
     http-request deny unless private
     mode http
+  <%- if_p("ha_proxy.stats_promex_enable") do -%>
+    http-request use-service prometheus-exporter if { path <%= p("ha_proxy.stats_promex_path") %> }
+    option http-use-htx
```

Comment:
This config option no longer exists as of haproxy/haproxy@9862787e8f4de3a901717565a6828fa3288919ba.
```suggestion
```